great_expectations/expectations/expectation.py

```diff
@@ -1305,7 +1275,15 @@ def validate_(  # noqa: PLR0913
 
     @property
     def configuration(self) -> ExpectationConfiguration:
-        return self._configuration
+        kwargs = self.dict(exclude_defaults=True)
```

Comment:
No need for defaults to be persisted if they'll be added back to the object upon deserialization